sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/ExceptionMappingPolicy.java

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.blob.models;
+
+import com.azure.core.http.HttpPipelineCallContext;
+import com.azure.core.http.HttpPipelineNextPolicy;
+import com.azure.core.http.HttpResponse;
+import com.azure.core.http.policy.HttpPipelinePolicy;
+import reactor.core.publisher.Mono;
+
+/**
+ * This is a exception mapping policy offers functionality to mapping the error to specific exception.
+ */
+public class ExceptionMappingPolicy implements HttpPipelinePolicy {
```

Comment:
Should we move `ExceptionMappingPolicy` types and `ScrubEtagPolicy` to implementation package? to avoid appearing them in Javadoc and to enable us to modify/delete it without breaking GAed lib.